controller/events.go

```diff
@@ -14,6 +14,8 @@ const (
 	EventReasonRebuilding       = "Rebuilding"
 	EventReasonFailedRebuilding = "FailedRebuilding"
 
+	EventReasonFailedRestoring = "FailedRestoring"
```

Comment:
We might want to use Nouns instead of Verbs
ie FailedRestore vs FailedRestoring, at the moment it's kinda inconsistent with FailedExpansion :)